src/main/groovy/life/qbic/datamodel/datasets/OxfordNanoporeMeasurement.groovy

```diff
@@ -112,6 +112,8 @@ final class OxfordNanoporeMeasurement {
     }
 
     private static String extractLibraryKit(String text) {
+        // cut off optional suffix
+        text = text.replace(":True", "")
```

Comment:
Where does the suffix come from? And why cut it off, would it be required maybe for the labs? This would then change the file name, right?

Reply:
we are not using it for anything and it was cut off before, just using a regex. now we have the cases where it is missing and where we need to cut it off, because the regex is not working for the version where it is already missing.